script/validators/source_validator.py

```diff
@@ -38,9 +38,11 @@
 # directory structure: peloton/scripts/formatting/<this_file>
 # PELOTON_DIR needs to be redefined if the directory structure is changed
 CODE_SOURCE_DIR = os.path.abspath(os.path.dirname(__file__))
-PELOTON_DIR = os.path.abspath(reduce(os.path.join, [CODE_SOURCE_DIR, os.path.pardir, os.path.pardir]))
+PELOTON_DIR = os.path.abspath(
+    reduce(os.path.join, [CODE_SOURCE_DIR, os.path.pardir, os.path.pardir])
+    )
 
-CLANG_FORMAT = "clang-format"
+CLANG_FORMAT = "clang-format-3.7"
```

Comment:
Please remove this change here. Currently I am checking how we will proceed with this.